tests/base/managers/entity.py

```diff
@@ -570,12 +570,23 @@ def test_mgr_entity_search_entities(entity_test_search):
     entities, search = entity_test_search
 
     assert search(u'e') == entities
-    assert search(u'entity string') == entities
+    assert search(u'entity string') == [entities[i] for i in [0, 1, 2, 3, 5, 6]]
```

Comment:
i think to make it clearer what the entity indeces translate to we probs need to add a comment with the strings